core/domain/topic_services.py

```diff
@@ -173,8 +173,8 @@ def _create_topic(committer_id, topic, commit_message, commit_cmds):
         additional_story_ids=topic.additional_story_ids,
         skill_ids=topic.skill_ids
     )
-    commit_cmds = [commit_cmd.to_dict() for commit_cmd in commit_cmds]
-    model.commit(committer_id, commit_message, commit_cmds)
+    commit_cmds_dict = [commit_cmd.to_dict() for commit_cmd in commit_cmds]
```

Comment:
This is not a dict. commit_cmd_dicts perhaps (plural indicates list).

Reply:
Done.